docs/tutorial/security.md

```diff
@@ -357,6 +352,32 @@ Content-Security-Policy: '*'
 Content-Security-Policy: script-src 'self' https://apis.mydomain.com
 ```
 
+### CSP HTTP Header
+
+Electron respects the [`Content-Security-Policy` HTTP header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy)
+which can be set using Electron's 
+[`webRequest.onHeadersReceived`](../api/web-request.md#webrequestonheadersreceivedfilter-listener)
+handler:
+
+```javascript
+const {session} = require('electron')
+
+session.defaultSession.webRequest.onHeadersReceived((details, callback) => {
+  callback({responseHeaders: `default-src 'self'`})
+})
+```
```

Comment:
I think you should point out that the best would be to start with `default-src 'none';` and then allow what's needed. It's always better to include than exclude. Most will probably then end up with:

```
default-src 'none';
script-src 'self';
img-src 'self' data:;
style-src 'self';
font-src 'self';
```

Reply:
That's a solid point, I agree with that.

Reply:
Updated to `default-src 'none'`